templates/webapi/test/overview.html.ep

```diff
@@ -152,7 +152,8 @@
                     </div>
                     <div class="col-2 align-self-center">
                          <%= help_popover('Help for the <em>Module</em> filter' => '
-                         <p>Shows jobs that contain the specified modules and allows also to filter them by the results.</p>
+                         <p><strong>Module by name</strong> - Shows jobs that contain the specified modules and allows also to filter them by the results.</p>
```

Comment:
The "and allows also to filter them by the results" does not relate to "Module by name" and deserves its own paragraph.